packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -59,9 +60,39 @@ export const getThemeWithVars = (
       shadow,
       palette,
     },
+    getColorSchemeSelector: () => '&',
```

Comment:
What's this?

Reply:
The `getColorSchemeSelector` added to the default theme is for the tests to work because this PR introduces some logic that uses `theme.getColorSchemeSelector()`.